### ISSUE TITLE
bug: python migrate fails on fresh local instance

### ISSUE THREAD

**Issue by introkun** [TreyWW/MyFinances#238]
### What version of MyFinances are you currently using?

v0.2.1

### What device type are you currently facing the issue on?

Desktop

### Describe the bug

```
❯ DEBUG=True python manage.py migrate

[BACKEND] Using local settings
[BACKEND] Using postgres database: myfinances_development
Traceback (most recent call last):
  File "/Users/sergey/opt/miniconda3/envs/py3122-myfinance/lib/python3.12/site-packages/django/db/backends/utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.UndefinedTable: relation "backend_featureflags" does not exist
LINE 1: ..."value", "backend_featureflags"."updated_at" FROM "backend_f...
```

**Comment by TreyWW** [comment]
Interesting, someone encountered this bug yesterday too. I re-installed everything today and didn't come across this though - odd.

Are you using v0.2.1 or the latest commit? if you're using v0.2.1 any chance you could upgrade to just the latest commit `main 60f360e` and see if it goes away?

**Comment by introkun** [comment]
I'm on the latest commit

**Comment by introkun** [comment]
No tables created due to this error

**Comment by TreyWW** [comment]
Oh yikes. It's due to the feature flag signal, but I just don't understand why as it's meant to be "post_migrate" which should run AFTER all other migrations. A potential fix could be to just comment out line 80

```py
post_migrate.connect(insert_initial_data)
```

**Comment by TreyWW** [comment]
But i'll try and reproduce this locally and find a fix. Sorry about that. Feel free to post any updates if anyone has any fixes

**Comment by introkun** [comment]
I've commented all urls in urlpatterns in file backend/urls.py and was able to call `python manage.py migrate`

**Comment by TreyWW** [comment]
Oh right.. That's a little odd that was a fix to the issue. I'll leave this issue open for me to work on. Good job on  fixing it though

**Comment by brentbundang** [comment]
Hey @TreyWW, I'm running into the same issue right now and the fix for me was to comment out the following in backend/urls.py if this helps you pinpoint the issue. 

https://github.com/TreyWW/MyFinances/blob/main/backend/urls.py#L25

**Comment by introkun** [comment]
<img width="1054" alt="image" src="https://github.com/TreyWW/MyFinances/assets/61635/bf029363-bb40-4309-85fb-c4b18d6c1b6f">
Problem is in this line.

**Comment by introkun** [comment]
DB is accessed during the app initialization, but DB is not yet ready

**Comment by TreyWW** [comment]
Hey @introkun,

Thanks for the comment, i'll look into this now, thanks for showing where the error originates from.

I'll let you know progress as it's made. Thanks :)

**Comment by TreyWW** [comment]
Should be fixed in [8284cee](https://github.com/TreyWW/MyFinances/commit/8284cee6f466c18948d977ba290ec8ce77a744c6)

Let me know if the error persists. Thanks :)

**Comment by introkun** [comment]
@TreyWW 
A similar issue happened again (git hash 8ea8798b634da797f7eeda9f5244dba26223bf0b, python ./manage.py migrate):
```
  File "/Users/sergey/opt/miniconda3/envs/py3122-myfinance/lib/python3.12/site-packages/django/urls/conf.py", line 39, in include
    urlconf_module = import_module(urlconf_module)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sergey/opt/miniconda3/envs/py3122-myfinance/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/Users/sergey/projects/MyFinances/backend/api/urls.py", line 11, in <module>
    path("invoices/", include("backend.api.invoices.urls")),
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sergey/opt/miniconda3/envs/py3122-myfinance/lib/python3.12/site-packages/django/urls/conf.py", line 39, in include
    urlconf_module = import_module(urlconf_module)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sergey/opt/miniconda3/envs/py3122-myfinance/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/Users/sergey/projects/MyFinances/backend/api/invoices/urls.py", line 3, in <module>
    from . import fetch, delete, edit, schedule, manage
  File "/Users/sergey/projects/MyFinances/backend/api/invoices/schedule.py", line 20, in <module>
    from infrastructure.aws.schedules.create_schedule import (
  File "/Users/sergey/projects/MyFinances/infrastructure/aws/schedules/create_schedule.py", line 10, in <module>
    from infrastructure.aws.iam.sfn import get_sfn_execute_role_arn
  File "/Users/sergey/projects/MyFinances/infrastructure/aws/iam/sfn.py", line 9, in <module>
    iam_client = get_iam_client()
                 ^^^^^^^^^^^^^^^^
  File "/Users/sergey/projects/MyFinances/infrastructure/aws/handler.py", line 62, in get_iam_client
    raise ValueError("If using schedules, the variables MUST be set.")
ValueError: If using schedules, the variables MUST be set.
```
